variables.tf

```diff
@@ -16,8 +16,62 @@
 
 variable "project_id" {
   description = "The project ID to deploy to"
+  type        = string
 }
 
-variable "bucket_name" {
-  description = "The name of the bucket to create"
+variable "network_regions" {
+  description = "List of regions and subnets to deploy VMware edge appliances"
+  type = set(object(
+    {
+      name        = string
+      inet_subnet = string
+      mgmt_subnet = string
+      lan_subnet  = string
+    }
+  ))
+  default = []
+}
+
+variable "cloud_router_asns" {
+  type    = list(number)
+  default = [65120, 65121, 65122, 65123]
+}
+
+variable "vce_asns" {
+  type    = list(number)
+  default = [65220, 65221, 65222, 65223]
+}
+
+variable "vce_machine_type" {
+  description = "GCP machine type for the Velocloud edge instance"
+  type        = string
+  default     = "n2-standard-4"
+
+}
+
+variable "velocloud_vco" {
+  description = "Base hostname to the Velocloud Orchestrator instance"
+  type        = string
+}
+
+variable "velocloud_token" {
+  description = "API token for the Velocloud Orchestrator instance"
```

Comment:
Question: do we have only one (prod) instance of Velocloud running, correct? Nonprod doesn't exist?

Reply:
We do not, just a single instance for now. 
We might have a second one, but I'll need to spend some time figuring that out.